start.sh

```diff
@@ -0,0 +1,117 @@
+#!/bin/bash
```

Comment:
What is this now used for? Why do we have `start.sh` and `docker-entrypoint.sh`?

Reply:
looks like som test file

Reply:
Yes, debug file. Removed